pylint/checkers/dunder_methods.py

```diff
@@ -167,6 +202,30 @@ def within_dunder_def(node: nodes.NodeNG) -> bool:
             parent = parent.parent
         return False
 
+    @only_required_for_messages("bad-dunder-name")
+    def visit_functiondef(self, node: nodes.FunctionDef) -> None:
+        """Check if known dunder method is misspelled or dunder name is not one
+        of the pre-defined names.
+        """
+        # ignore module-level functions
+        if not node.is_method():
+            return
+
+        # Detect something that could be a bad dunder method
+        if (
+            node.name.startswith(("_", "__"))
```

Comment:
```suggestion
            node.name.startswith("_")
```
After thinking about it, this is enough as if this is true the other one will be too.

Reply:
lol 🤦‍♀️ of course, I missed this as well. I put that initially :)